web/src/ImpactDetailMode.svelte

```diff
@@ -68,6 +82,19 @@
     </p>
 
     <PrevNext list={routes} bind:idx />
+
+    {#if routes[idx][0] == null}
+      <p style:color="red">
+        No possible route before changes (
```

Comment:
I'll remove the "usually a bug" after fixing this case. There could be legit cases where a user manages to disconnect some roads after changes.